src/main/java/org/ecommerce/util/CrudOperations.java

```diff
@@ -3,10 +3,8 @@
 import java.util.List;
 
 public interface CrudOperations<T> {
```

Comment:
Please use generics for the id data type

Reply:
https://github.com/algutierrez-griddynamics/e-commerce/pull/11/commits/01e81bdbd914d57659ee275a2dcc76bea1c49f4c done